subsys/bluetooth/host/keys.h

```diff
@@ -96,16 +96,25 @@ enum {
 
 struct bt_keys_link_key {
 	bt_addr_t		addr;
+	u8_t                    storage_start[0];
```

Comment:
We should put __aligned(sizeof(void*)) here.
So that if we modify the struct before the storage_start we don't end up changing struct padding in the storage area.